### PR TITLE
Add option to run a single chromosome for stats tests

### PR DESCRIPTION
**Pull request by jkgoodrich**
I needed to make some modifications to get the stats to run without memory errors.

This PR also includes:
- Setting NaN values to NA for the stats
- Not running the stats test if AN is 0 for either the exomes or genomes
- Remove faf95 from grpmax if it exists

### CODE REVIEW

File: gnomad_qc/v4/create_release/create_combined_faf_release_ht.py [hunk 1]
```diff
@@ -405,7 +413,7 @@ def perform_contingency_table_test(
             hl.is_defined(x[0]) & hl.is_defined(x[1]),
             hl.bind(
                 lambda f1, f2: hl.or_missing(
-                    (f1.AC > 0) | (f2.AC > 0),
+                    (f1.AC > 0) | (f2.AC > 0) & (f1.AN > 0) & (f2.AN > 0),
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/create_release/create_combined_faf_release_ht.py:416]: Could you update L409 to just include that theres coverage for both?

File: gnomad_qc/v4/create_release/create_combined_faf_release_ht.py [hunk 2]
```diff
@@ -576,21 +597,52 @@ def get_combine_faf_resources(
...
             "contingency_table_ht": get_freq_comparison(
                 "contingency_table_test", test=test, filtered=filtered
             )
-        },
-        pipeline_input_steps=[combined_frequency],
+        }
+
+    if stats_combine_all_chr:
+        cmh_resources["input_resources"] = {
+            "--perform-cochran-mantel-haenszel-test --stats-chr (all chr)": {
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/create_release/create_combined_faf_release_ht.py:624]: Small thing but maybe 
```suggestion
            "--perform-cochran-mantel-haenszel-test --stats-chr (any chr)": {
```
At first pass, this was a bit confusing as I though it was suggesting a list of all chrs

**Review by mike-w-wilson — APPROVED** [review]
a few nit picks but LGTM!

File: gnomad_qc/v4/create_release/create_combined_faf_release_ht.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/create_release/create_combined_faf_release_ht.py:416]: sorry, I'm not positive what you are suggesting, can you add the suggested code change?

[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_qc/v4/create_release/create_combined_faf_release_ht.py:416]: I'm going to merge and put in the next PR, but let me know in that PR what to change for this

**Pull request merged**